content/develop/interact/search-and-query/advanced-concepts/vectors.md

```diff
@@ -9,369 +9,455 @@ categories:
 - oss
 - kubernetes
 - clients
-description: Learn how to use vector fields and vector similarity queries
+description: Learn how to use vector fields and perform vector searches in Redis
 linkTitle: Vectors
 math: true
 title: Vectors
 weight: 14
 ---
 
-*Vector fields* allow you to use vector similarity queries in the [`FT.SEARCH`]({{< baseurl >}}/commands/ft.search/) command.
-*Vector similarity* enables you to load, index, and query vectors stored as fields in Redis hashes or in JSON documents (via integration with the [JSON module]({{< relref "/develop/data-types/json/" >}}))
+Redis is commonly used as a [highly-performant vector database](https://redis.io/blog/benchmarking-results-for-vector-databases/). Vector fields enable you to perform semantic search based on vector embeddings in combination with text, numerical, geospatial, or tag metadata.
 
-Vector similarity provides these functionalities:
+**Just looking to get started?** Checkout the vector [quickstart guide]({{< baseurl >}}/develop/get-started/vector-database) and the [Redis AI Resources](https://github.com/redis-developer/redis-ai-resources) repo for more assistance.
 
-* Realtime vector indexing supporting two indexing methods:
 
-    - FLAT - a simple, lightweight index.
+## Overview
 
-    - HNSW - Modified version of [nmslib/hnswlib](https://github.com/nmslib/hnswlib), which is an implementation of [Efficient and robust approximate nearest neighbor search using Hierarchical Navigable Small World graphs](https://arxiv.org/ftp/arxiv/papers/1603/1603.09320.pdf).
+1. [**Create a vector index**]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#create-a-vector-index): Redis maintains a secondary index on top of your data with a defined schema (including vector fields and metadata). Redis supports [`FLAT`]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#flat-index) and [`HNSW`]({{< baseurl >}}/develop/interact/search-and-query/advanced-concepts/vectors#hnsw-index) vector index types.
```

Comment:
this overview is much more clear!

Reply:
only note: might change "and metadata filters" to "and flexible search over all additional fields in HASH or JSON" 

My thought is this helps highlight the flexibility of those datatypes as a feature instead of a quirk to redis if that makes sense